submissions/TaniaBondarenko/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Movie-seat-booking</title>
+    <link href="styles.css" type="text/css" rel="stylesheet" />
+  </head>
+  <body>
+    <main class="mainContainer">
+      <section class="posterContainer">
+        <div class="poster"><img class="posterImg" src="images/brw.jpg" alt="Poster to film" /></div>
+        <div class="badge" data-description="lux">
+          <div class="badgeSeat lux"></div>
+          <div class="ticketType">
+            <div class="name"><a>Lux</a></div>
+            <span class="price">100</span>
+            <small class="currency">usd</small>
+          </div>
+
+          <div class="badgeSeat vip"></div>
+          <div class="ticketType">
+            <div class="name"><a>Vip</a></div>
+            <span class="price">300</span>
+            <small class="currency">usd</small>
+          </div>
+        </div>
+        <div class="legend">
+          <div class="occupiedSeat"></div>
+          <div class="description">
+            <a>Occupied<br />seat</a>
+          </div>
+          <div class="blockedSeat"></div>
+          <div class="description">
+            <a>Blocked<br />seat</a>
+          </div>
+        </div>
+      </section>
+
+      <div class="cinemaContainer">
+        <div class="filmInfo">
+          <h2 class="filmTitle">2001: A Space Odyssey</h2>
+          <br />
+          <p>2h 29min</p>
+          <p>Adventure, Sci-Fi</p>
+          <p>IMDb 8.3</p>
+          <time datetime="1968-05-12"> 12 May 1968</time><br />
+        </div>
+        <div class="screenImg">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="-290 -20 1400 50">
+            <path
+              d="M3.2,20l-2,0.2l-0.3-4l2-0.2C136.2,5.3,269.6,0,403,0s266.8,5.3,400.2,16l2,0.2l-0.3,4l-2-0.2 C669.5,9.3,536.3,4,403,4S136.4,9.3,3.2,20z"
+            ></path>
+          </svg>
+        </div>
+        <div class="screenText">SCREEN</div>
+
+        <div class="seats">
+          <br />
+          <section class="rowLux">
+            <span class="rowText">1</span>
+            <input id="checkbox1-6" type="checkbox" value="1-6" class="hiddenCheckbox" tabindex="-1" />
+            <button class="seat"></button>
```

Comment:
Why you use `input type="checkbox"` and `button` (two interractive elemnts) for one seat?

Reply:
I've done the same in the html popup menu. Should I replace `button` with `div`?

Reply:
You can use `div` or `span` etc. I mean you already have an interactive element you don't need any other interactive elements. When user will click on the seat from which element you will handle click?)

Reply:
>  When user will click on the seat from which element you will handle click?)
I haven't thing about yet...

Reply:
Replaced `button` with `span`